apps/web/src/components/map/MapView.tsx

```
@@ -1,8 +1,9 @@
 'use client';
 
-import { useRef, useCallback } from 'react';
+import { useRef, forwardRef, useImperativeHandle } from 'react';
 import { Map, GeolocateControl, Marker } from 'react-map-gl/mapbox';
```

Biome (2.3.13)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)